### ISSUE TITLE
P03-C01 / P03-C02 - Moving Loss / Hyperparameters

### ISSUE THREAD

**Issue by madanvikram** [zackchase/mxnet-the-straight-dope#110]
I have two overall comments:

1) It would helpful to explain the moving loss calculation and why it is important. The moving loss calculation is not entirely intuitive and may seem arbitrary to beginners. I think it is important to frame it from the perspective that it helps SGD. In addition, here we've focused on a momentum based approach, but I think it would be good to introduce learning rate decay as well (we should also explain the learning rate).

2) It would be helpful to understand how to choose the appropriates value for the hyperparameters introduced (e.g., learning rate and smoothing constant). It is difficult choosing the appropriate value for your hyperparameters, and here they are chosen for you without any explanation to why they make sense. This is especially confusing when you use different learning rates when implementing the neural network using Gluon versus defining from scratch (0.1 versus 0.001).

**Comment by mli** [comment]
gluon version uses a smaller learning rate because its gradients are divided by the batch size, which is specified by `trainer.step(batch_size)`
